### PR TITLE
Magnifier: Add the option to save captures

### PR DESCRIPTION
**Pull request by gigaroby**

### CODE REVIEW

**Review by vkoskiv — COMMENTED** [review]
Added comments. Just some nitpicks, I'm fine with this being merged as-is. 8)

**Comment by vkoskiv** [comment]
@linusg Can we get this merged? No conflicts and I'm fine with this being merged as-is

**Comment by gigaroby** [comment]
Sorry, I was on vacation. There's now support for saving in different formats. I don't think the UX for this is great (if users type the wrong extension all they get is a "Unable to save") but we can fix that later.

File: Userland/Applications/Magnifier/main.cpp [hunk 1]
```diff
@@ -4,24 +4,44 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
+#include "LibGUI/MessageBox.h"
+#include "LibGfx/BMPWriter.h"
```
Comment (AtkinsSJ) [inline Userland/Applications/Magnifier/main.cpp:8]:
```suggestion
#include <LibGUI/MessageBox.h>
#include <LibGfx/BMPWriter.h>
```

File: Userland/Applications/Magnifier/main.cpp [hunk 2]
```diff
@@ -40,6 +60,28 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         app->quit();
     })));
 
+    TRY(file_menu->try_add_action(GUI::CommonActions::make_save_as_action([&](auto&) {
+        // filename is filled-in by save_bitmap_to and used for warnln to give a more descriptive error
```
Comment (AtkinsSJ) [inline Userland/Applications/Magnifier/main.cpp:64]: This comment refers to `save_bitmap_to` which doesn't exist. Please remove it.

**Review by AtkinsSJ — CHANGES_REQUESTED** [review]
I have a couple of nit-picks, otherwise this looks good.

**Review by AtkinsSJ — APPROVED** [review]

**Pull request merged**